src/main/java/webApp/webController.java

```diff
@@ -53,6 +54,8 @@ public Response createSurvey(@RequestBody SurveyMessage surveyMessage) {
                     return new Response(null, "error",
                             "Min is greater than max for question \"" + questionMsg.getQuestion() + "\"");
                 }
+            } else if (questionMsg.getType().equals(dropdownQuestionType)) {
+                questionList.add(new DropdownQuestion(questionMsg.getQuestion(), (ArrayList<String>) questionMsg.getOptions()));
```

Comment:
If you are always casting to an ArrayList<String> then shouldn't `getOptions` just return an ArrayList<String>

Reply:
Otherwise if someone gets the wrong type in there this will crash.

Reply:
Good catch, on it.